includes/class-amp-story-post-type.php

```diff
@@ -350,6 +350,18 @@ public static function remove_classic_editor_link( $actions, $post ) {
 	 * @return string[] Filtered array of allowed CSS attributes.
 	 */
 	public static function filter_safe_style_css( $attr ) {
+
+		// Only continue if it's a REST request for amp_story post type.
+		if ( ! isset( $_REQUEST['rest_route'] ) || empty( $_REQUEST['rest_route'] ) ) {
```

Comment:
Untested, but this might cause issues with WP-CLI because `$_REQUEST` does not exist in a CLI context.

`rest_api_loaded()` in core uses `$GLOBALS['wp']->query_vars['rest_route']` to get the route. Maybe that can be used too?

Reply:
Good point, will check now.

Reply:
Changed within 528a32c.